### PR TITLE
Multiclet merge

### PR DESCRIPTION
**Pull request by AntonKozlov**

### CODE REVIEW

**Comment by AntonKozlov** [comment]
Travis failed to build arm templates, seems like CROSS_COMPILE was not used (everywhere bare `gcc` was called).
I have no such problem on my local machines. Can someone check this branch compiles with arm templates at least?

**Comment by anton-bondarev** [comment]
I havn't any problem on my machine with building this templates

File: mk/flags.mk [hunk 1]
```diff
@@ -10,16 +10,16 @@ LDFLAGS ?=
...
+CC      ?= $(CROSS_COMPILE)gcc
+CPP     ?= $(CC) -E
+CXX     ?= $(CROSS_COMPILE)g++
+AR      ?= $(CROSS_COMPILE)ar
+AS      ?= $(CROSS_COMPILE)as
+LD      ?= $(CROSS_COMPILE)ld
+NM      ?= $(CROSS_COMPILE)nm
+OBJDUMP ?= $(CROSS_COMPILE)objdump
+OBJCOPY ?= $(CROSS_COMPILE)objcopy
+SIZE    ?= $(CROSS_COMPILE)size
```
Comment (abusalimov) [inline mk/flags.mk]: Travis fails because of these changes

Comment (AntonKozlov) [inline mk/flags.mk]: Have you any clue how it affects and why we can't reproduce it on archlinux and mint?

Comment (abusalimov) [inline mk/flags.mk]: Yep, see Travis [logs](https://travis-ci.org/embox/embox/jobs/74988530#L92):

    $ export CC=gcc

That's how Travis [works](http://docs.travis-ci.com/user/languages/c/#Choosing-compilers-to-test-against) for C projects.

> Testing against two compilers will create (at least) 2 rows in your build matrix. For each row, Travis CI C builder will export the `CC` env variable to point to either `gcc` or `clang`.

File: third-party/bsp/multiclet/libs/source/uart/uart_dm2uart.c [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+			*iterator2 = *iterator2 - 1;
+			return 0;
+		}
+		else
+		{
+			return 1;
+		}
+}
+/**
+  * @brief  Проверка на заполнение буфера fifo прочитанными данными (проверка прихода 32 элементов).
```
Comment (alexkalmuk) [inline third-party/bsp/multiclet/libs/source/uart/uart_dm2uart.c]: Russian detected!

**Pull request merged**